src/ripple/net/impl/RPCCall.cpp

```diff
@@ -497,14 +497,26 @@ class RPCParser
 
     bool isValidJson2(Json::Value const& jv)
     {
+        if (jv.isArray())
+        {
+            if (jv.size() == 0)
```

Comment:
Why is valid json2 array invalid if it contains an empty array?

Reply:
It comes from here: http://www.jsonrpc.org/specification#batch

It would probably be a good idea for me to put this link in as a comment.